ms-windows/osgeo4w/package.cmd

```diff
@@ -28,7 +28,7 @@ if "%PACKAGENAME%"=="" goto usage
 if "%ARCH%"=="" goto usage
 if not "%SHA%"=="" set SHA=-%SHA%
 if "%SITE%"=="" set SITE=qgis.org
-if "%BUILDNAME%=="" set BUILDNAME=%PACKAGENAME%-%VERSION%%SHA%-Release-VC14-%ARCH%

```

Comment:
Was the lineending change on purpose?

Reply:
looks like an artifact from git - package.cmd has CRLF endings.